lib/ansible/modules/network/f5/bigip_virtual_server.py

```diff
@@ -250,7 +250,7 @@ def set_rules(api, name, rules_list):
         return False
     rules_list = list(enumerate(rules_list))
     try:
-        current_rules = map(lambda x: (x['priority'], x['rule_name']), get_rules(api, name))
+        current_rules = list(map(lambda x: (x['priority'], x['rule_name']), get_rules(api, name)))
```

Comment:
Most of the time if you want a list and you're using map() you can convert to a list comprehension for a more pythonic (and faster, although that's almost never an issue) implementation:

``` python
current_rules = [(x['priority'], x['rule_name']) for x in get_rules(api, name)]
```